extensions/in-memory/policy-store-memory/src/main/java/org/eclipse/dataspaceconnector/policy/store/memory/InMemoryPolicyStore.java

```diff
@@ -0,0 +1,82 @@
+package org.eclipse.dataspaceconnector.policy.store.memory;
```

Comment:
nit: copyright header is missing from the newly created files.

Reply:
added